vcpkg/produce/package-ado-repository.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Tutorial: Package a dependency from an Azure DevOps repository"
+description: This tutorial shows how to create a vcpkg port to package a dependency from an Azure DevOps repository.
+author: vicroms
+ms.author: viromer
+ms.date: 10/24/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+
+#CustomerIntent: As a vcpkg user, I want to package a dependency hosted in an Azure DevOps repository
+---
+
+# Tutorial: Package a dependency from an Azure DevOps repository
+
+This tutorial shows you how to create a vcpkg port to package a dependency
+hosted in an Azure DevOps repository. We recommend that you read the [Tutorial:
+Create and publish packages](../get_started/get-started-packaging.md) article
+before proceeding.
+
+In this tutorial we show you how to:
+
+> [!div class="checklist"]
+> * [Create a port directory](#1---create-a-port-directory)
+> * [Create the port's manifest](#2---create-the-ports-manifest)
+> * [Create the port's recipe script](#3---create-the-ports-recipe-script)
+
+## Prerequisites
+
+* A terminal
+* A C++ copmpiler
+* vcpkg
+* Git
+* A library hosted in an Azure DevOps repository
+
+## 1 - Create a port directory
+
+Create a new directory with the name you want to give to your port.
+
+## 2 - Create the port's manifest
+
+Inside the port's directory create a file named `vcpkg.json`, replace the
+placeholders with the appropiate values for your package.
```

Comment:
Maybe the reader should be instructed to run "vcpkg new" and guided about what changes to make to the file after? I think we want users to establish a habit of using vcpkg commands wherever possible to do things automatically. While it makes little difference in a tutorial where they can copy and paste, it's a useful habit to have if they want to do it again at a later time.